addon-test-support/helpers.js

```diff
@@ -168,42 +168,47 @@ export function keyEvent(selector, type, keyCode) {
 }
 
 /*
-  The find test helper uses `querySelectorAll` to search inside the test
+  The find test helper uses `querySelector` to search inside the test
   DOM (based on app configuration for the rootElement).
 
   Alternalively, a second argument may be passed which is an element as the
   DOM context to search within.
 
   @method find
-  @param {String|HTMLElement|NodeList} CSS selector to find one or more elements in the test DOM
+  @param {String} CSS selector to find one or more elements in the test DOM
   @param {HTMLElement} contextEl to query within, query from its contained DOM
-  @return {null|HTMLElement|NodeList} null, one element or a (non-live) list of element objects
+  @return {null|HTMLElement} null or an element
   @public
 */
 export function find(selector, contextEl) {
   let result;
-  if (selector instanceof HTMLElement || selector instanceof NodeList) {
-    result = selector;
-  } else if (contextEl instanceof HTMLElement) {
-    result = contextEl.querySelectorAll(selector);
+  if (contextEl instanceof HTMLElement) {
+    result = contextEl.querySelector(selector);
   } else if (Object.prototype.toString.call(selector) === "[object String]") {
-    result = document.querySelectorAll(`${settings.rootElement} ${selector}`);
+    result = document.querySelector(`${settings.rootElement} ${selector}`);
   }
-  return (result.length === 0) ? null : (result.length === 1) ? result[0] : result;
+  return result;
 }
 
-
 /*
-  Since the find helper may return a NodeList or an HTMLElement the first helper
-  only returns the first matching element
+  The findAll test helper uses `querySelectorAll` to search inside the test
+  DOM (based on app configuration for the rootElement).
+
+  Alternalively, a second argument may be passed which is an element as the
+  DOM context to search within.
 
-  @method first
-  @param {String} CSS selector to find one element in the test DOM
+  @method findAll
+  @param {String} CSS selector to find elements in the test DOM
   @param {HTMLElement} contextEl to query within, query from its contained DOM
-  @return {HTMLElement} first element found with matching selector
+  @return {NodeList} A (non-live) list of zero or more HTMLElement objects
   @public
```

Comment:
I believe that a NodeList containing just HTMLElements is technically an `HTMLCollection`, but I get lost on the official DOM classes.